src/generic/listctrl.cpp

```diff
@@ -1177,14 +1177,13 @@ void wxListHeaderWindow::OnPaint( wxPaintEvent &WXUNUSED(event) )
         // if we have an image, draw it on the right of the label
         if ( imageList )
         {
-            imageList->Draw
-                       (
-                        image,
-                        dc,
-                        xAligned + wLabel - ix - HEADER_IMAGE_MARGIN_IN_REPORT_MODE,
-                        HEADER_OFFSET_Y + (h - iy)/2,
-                        wxIMAGELIST_DRAW_TRANSPARENT
-                       );
+            wxBitmap bmp = imageList->GetImageBitmapFor(this, image);
+            dc.DrawBitmap(
+                          bmp,
+                          xAligned + wLabel - ix - HEADER_IMAGE_MARGIN_IN_REPORT_MODE,
+                          HEADER_OFFSET_Y + (h - iy)/2,
+                          wxIMAGELIST_DRAW_TRANSPARENT
```

Comment:
Just for the record: I'm removing this, it's wrong to use it as a `bool useMask`, and adding `true` here and to all the other places where `DrawBitmap()` is called instead.